src/lang/bg-BG.json

```diff
@@ -844,26 +844,5 @@
   "vision_all_time":"All time",
   "vision_placed_observer":"placed Observer at",
   "vision_placed_sentry":"placed Sentry at",
-  "vision_ward_log":"Ward Log",
-  "region_0":"Автоматичен",
```

Comment:
@albertcui why are these strings getting removed?  Should we close and reopen the PR from crowdin?